index.js

```diff
@@ -66,3 +66,474 @@ hamburguerMenuIcon.addEventListener('click', () => {
+        fill="#505F79"
+      />
+    </svg>
+  </button>
+  </div>
+  </div>
+</div>
+</article>
+<span class="modal-card__span"></span>
+`;
+
+  const corssIconPopUpCard = document.querySelector('.modal-card__cancel');
+  corssIconPopUpCard.addEventListener('click', () => {
+    // console.log('cross working');
+    window.location.reload();
+  });
+});
+
+const buttonProject2 = document.querySelector('.see-project2');
+const body3 = document.querySelector('body');
+buttonProject2.addEventListener('click', () => {
+  // console.log('hello');
+  body3.innerHTML = `
+<article class="modal-card">
+<h2 class="modal-card__h2">
+  ${projectListObject[1].name}
+  <svg
+    class="modal-card__cancel"
+    width="24"
+    height="24"
+    viewBox="0 0 24 24"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
+      d="M5.29289 5.29289C5.68342 4.90237 6.31658 4.90237 6.70711 5.29289L12 10.5858L17.2929 5.29289C17.6834 4.90237 18.3166 4.90237 18.7071 5.29289C19.0976 5.68342 19.0976 6.31658 18.7071 6.70711L13.4142 12L18.7071 17.2929C19.0976 17.6834 19.0976 18.3166 18.7071 18.7071C18.3166 19.0976 17.6834 19.0976 17.2929 18.7071L12 13.4142L6.70711 18.7071C6.31658 19.0976 5.68342 19.0976 5.29289 18.7071C4.90237 18.3166 4.90237 17.6834 5.29289 17.2929L10.5858 12L5.29289 6.70711C4.90237 6.31658 4.90237 5.68342 5.29289 5.29289Z"
+      fill="#67798E"
+    />
+  </svg>
+</h2>
+
+<ul class="modal-card__list">
+  <li class="modal-card__canopy">${projectListObject[1].company}</li>
+  <li class="modal-card__role">
+    <svg
+      width="8"
+      height="8"
+      viewBox="0 0 8 8"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+    </svg>
+    ${projectListObject[1].role}
+  </li>
+  <li class="modal-card__year">
+    <svg
+      width="8"
+      height="8"
+      viewBox="0 0 8 8"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <circle cx="4" cy="4" r="4" fill="#C1C7D0" />
+    </svg>
+    ${projectListObject[1].year}
+  </li>
+</ul>
+<img class="modal-card__img" src="${projectListObject[1].featuredImage}" alt="" />
+<p class="modal-card__p">
+${projectListObject[1].description}
+</p>
+<ul class="modal-card__lits-2">
+  <li class="modal-card__html modal-card__li">${projectListObject[1].technologies[0]}</li>
+  <li class="modal-card__css modal-card__li">${projectListObject[1].technologies[1]}</li>
+  <li class="modal-card__javascript modal-card__li">${projectListObject[1].technologies[2]}</li>
+</ul>
```

Comment:
- [ ] Kindly 😊 try to fix the modal on the mobile version as you see here the content of the modal is on the image, so please try to fix that to make it like the Figma 💯
- [ ] Also consider removing the black line which appears in your design because it is not in the Figma, so please let's fix that to be like the Figma 😃
- [ ] Please try to add the hr line above the two buttons to be like the Figma 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
| ![image](https://user-images.githubusercontent.com/83360636/223809907-f87edc6e-8c7d-4047-8cac-422fa9db96bf.png) |    ![image](https://user-images.githubusercontent.com/83360636/223808836-f5416fb0-4415-4ce3-aa19-4dc6a51d5539.png) |

Reply:
Hello @osamaashraf6,

Regarding the second request, the black bold line is on the figma's design. Just for now I'm not going to remove it so maybe you or the next reviewer can take a second look at that:

- Figma's design
![image](https://user-images.githubusercontent.com/109441277/223846539-d7eccb1d-58c8-4b0c-9154-6bfb9c9b2f80.png)

Reply:
Hello @osamaashraf6,

Regarding the 2nd request. Could you kindly take a second look at the figma's design?. I see there is a black bold horizontal line at the end of the card:

- Figma's design
![image](https://user-images.githubusercontent.com/109441277/223851436-4a7fc6d5-e441-4d7b-8a8c-97373b13fe57.png)

Thanks for your review, the other points were corrected 😃